zilliqa/src/p2p_node.rs

```diff
@@ -255,12 +251,37 @@ impl P2pNode {
                         SwarmEvent::NewListenAddr { address, .. } => {
                             info!(%address, "P2P swarm listening on");
                         }
+                        // this is necessary - https://docs.rs/libp2p-kad/latest/libp2p_kad/#important-discrepancies
+                        SwarmEvent::Behaviour(BehaviourEvent::Identify(identify::Event::Received { peer_id, info, .. })) => {
+                            debug!(%peer_id, ?info, "identify event");
```

Comment:
This is already logged on line 249 (and the same applies for the other new `debug!` logs)